src/pug/blog/year_one_recap.pug

```diff
@@ -0,0 +1,67 @@
+extends _blog.pug
+
+block append subtitle
+  | Year one recap
+
+block content
+  h2 Year one recap
+  +news_header('22 November 2023', 'report_18', '')
+
+  p One year ago Input Labs went live, it is time to review our achievements and our failures.
+
+  h3 Things that went nicely
+  h4 Community
+  ul
+    li Our nice #[em community kept growing], now with 735 discord members. This is by far the thing that makes us more proud, and it is awesome to see people helping each other there. To make it even better, some of the members are experts in their fields (electrical engineers, industrial designers) and contribute with expert advice and knowledge.
+    li We got a #[em small office] in the #[+ax('https://maria.io/about/') Maria 01 startup campus] in Helsinki, and we moved our 3D-printers and other tools there, making it the "Lab" of Input Labs.
+    li The project received plenty of #[em positive feedback] in the different social channels in which we have presence, and lot of people encouraged us to continue. We got very limited (almost negligible) negative interactions.
+    li About #[em 30 community mods] were created for the Alpakka controller, most of them modifications of the 3D-printed models for different purposes.
+    li Our project got supported by #[em 10 wonderful folks in Patreon].
+    li We opened the #[em distribution channel (shop)] to better serve testers, reviewers, and everyone without the tools to build a controller from scratch. Setting up the logistics to build and deliver all these components has been a challenge, but also a success in verifying that the "DIY-or-buy" model for supporting open source projects can work for us.
```

Comment:
Needs to be rephrased. It is not clear to what the 2nd sub-sentence refers. E.g. "We opened the #[em distribution channel (shop)] to better serve testers, reviewers, but also for people who do not have access to all the materials and tools, necessary to build their own Alpakka"